### DISCUSSION TITLE
Clone cleaner, turns itself on by itself

### DISCUSSION THREAD

**Discussion by bobpuffer1** [artyfacialintelagent/CloneCleaner#7] [Q&A]
I initially put this in issues before there was a discussion forum:

- turn clone cleaner on, used for a while
- turn clone cleaner off
- generate some more images
- notice that clone cleaner has turned itself back on

**Comment by artyfacialintelagent** [comment]
I don't see how it could it turn on "by itself", except upon startup (that's an issue which hopefully will be fixed by v1.0). But the enabled/disabled state is saved in image metadata. If you load an image generated by CloneCleaner using e.g. the image browser extension and then send it to txt2img, then all CloneCleaner settings will load too - and as a consequence enable CloneCleaner for the next generation.

**Reply by bobpuffer1** [reply]
I run infinite image browsing, but I run it as a standalone outside of automatic 1111 so I don't see that causing this problem I'll continue to watch it just to make sure that I haven't made a mistake.

**Reply by bobpuffer1** [reply]
It did it again. Went to sleep with it off. Came back this morning and it was on. Mysterious. I've turned it off in both T2I and I2I because I use dynamic prompts and it conflicts with them. And I was waiting to use a fresh prompt so I can try out clone cleaner.

**Reply by bobpuffer1** [reply]
And it did it again. I don't think I'm delusional. I think this is a real problem. One thing I did differently today was to use the extras tab which I don't usually do.

**Reply by bobpuffer1** [reply]
This problem seems to have to do with switching between T2I and I2I. There may be extras tab in the loop too. I have clone cleaner turned off everywhere because I use dynamic prompts and it definitely turns itself back on.

**Reply by artyfacialintelagent** [reply]
Thanks for the updates. I haven't seen this problem yet myself but I don't doubt that it's happening to you. So I really appreciate you trying to pin down the problem like this. I'll try switching back and forth between txt2img and img2img too to see if I can trigger it. But please continue the updates as you learn more!

**Comment by bobpuffer1** [comment]
Interestingly, enough. I haven't had it happen a single time today. I've been at work for about five hours so far and have generated 500 images and not a single time has clone cleaner turned itself on.
